src/constants/exchangeConstants.js

```diff
@@ -39,4 +39,5 @@ export const SLOW = 'min';
 export const NORMAL = 'avg';
 export const FAST = 'max';
 
-export const UNISWAP_GRAPH_ID = 'QmXKwSEMirgWVn41nRzkT3hpUBw29cp619Gx58XW6mPhZP';
+// mainnet only
+export const UNISWAP_SUBGRAPH_NAME = 'uniswap/uniswap-v2';
```

Comment:
Why is it "mainnet only"? When using ID, it worked just fine on ropsten.

Reply:
Per our discussion in DM – each subgraph represents single network, however, in this case you're fetching only symbols from this subgraph so it's ok as long as we're not fetching something more network related like contract addresses, etc.